src/Cache/Cache.php

```diff
@@ -98,7 +98,7 @@ public static function fromJson($json)
             ));
         }
 
-        $requiredKeys = array(
+        static $requiredKeys = array(
```

Comment:
please provide reasoning.
it has value for older php, but currently if an array is not modified, it's declared only once in memory